pkg/webui/console/views/device-add/manual/form/advanced-settings.js

```diff
@@ -87,6 +89,7 @@ const AdvancedSettingsSection = props => {
     deviceClass === DEVICE_CLASS_MAP.CLASS_B_C || deviceClass === DEVICE_CLASS_MAP.CLASS_B
   const isClassC =
     deviceClass === DEVICE_CLASS_MAP.CLASS_B_C || deviceClass === DEVICE_CLASS_MAP.CLASS_C
+  const disableFactoryPresetFreq = isAuBand(freqPlan) || isCnBand(freqPlan) || isUsBand(freqPlan)
```

Comment:
Ah ok so here `freqPlan` is passed to a function that takes `band`. Let's not mix these terms because they're not the same.

Reply:
So here basically it would be `const disableFactoryPresetFreq = hasCFListTypeChMask(freqPlan)` which is quite elegant and makes it easier to add more bands to that list later